R/install.R

```diff
@@ -111,32 +133,25 @@ repository <-
         version = BiocManager::version(),
         binary_base_url = BINARY_BASE_URL)
 {
-    ## are we running on a docker container?
-    platform <- ""
-    bioconductor_docker_version <- Sys.getenv("BIOCONDUCTOR_DOCKER_VERSION")
-    if (!nzchar(bioconductor_docker_version)) {
-        platform <- Sys.getenv("TERRA_R_PLATFORM")
-        bioconductor_docker_version <-
-            Sys.getenv("TERRA_R_PLATFORM_BINARY_VERSION")
-    }
-    if (!nzchar(bioconductor_docker_version))
-        return(character())
+    platform_docker <- .platform_bioc_docker_version()
 
-    ## is the docker container configured correctly?
+    bioconductor_docker_version <-
+        platform_docker[["bioconductor_docker_version"]]
+    platform <- platform_docker[["platform"]]
```

Comment:
I think these should be `platform_docker$bioconductor_docker_version`, to be consistent with the way, e.g., `bioconducter_version$major` is used?